fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: 
+      url('Montserrat-Bold.woff2') format('woff2'),
+      url('Montserrat-Bold.woff') format('woff');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: 
+      url('Montserrat-Medium.woff2') format('woff2'),
+      url('Montserrat-Medium.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: 
+      url('Montserrat-Regular.woff2') format('woff2'),
+      url('Montserrat-Regular.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/LOCO__Oxana/pullRequest?prid=5565108)